sysutils/borgbackup/Portfile

```diff
@@ -45,12 +47,23 @@ depends_lib-append  path:lib/libssl.dylib:openssl \
 depends_run-append  port:py${python.version}-msgpack \
                     port:py${python.version}-packaging
 
+# Run tests, but skip test_unix_socket, since it will fail because the path is
+# longer than the maximum length of a unix socket name when run in a MacPorts
+# build.
+depends_test-append port:py${python.version}-pytest-xdist \
+                    port:py${python.version}-pytest-benchmark
+
+test.run            yes
+test.args-append    -k 'not test_unix_socket' \
+                    -n ${build.jobs} \
+                    --pyargs borg.testsuite
+
 post-build {
     reinplace "s|sphinx-build|sphinx-build-${python.branch}|g" \
```

Comment:
While we're here, could you fix this `reinplace` so that if the build fails and is then retried, this replacement does not occur a second time and cause a new and different failure? This would fix https://trac.macports.org/ticket/72139.